### ISSUE TITLE
Integrate StopFinder with OBA

### ISSUE THREAD

**Issue by caitbonnar** [OneBusAway/onebusaway-iphone#213]
Here at UW, there's been ongoing research on [enhancing independence and safety for blind and deaf-blind public transit riders](http://lazowska.cs.washington.edu/p3247-azenkot.pdf).  One of the projects we've been working on recently is a proposed extension of OneBusAway called [StopFinder](http://src.acm.org/2012/PrasainSanjana.pdf) that involves two main components: first, the ability to look up landmark information about bus stops (e.g. sign position and whether it is standalone or on a pole, whether there is a shelter, etc.) through a non-graphical interface so that blind riders can more easily determine location of stops; second, the collection of this information through crowdsourcing, which should involve both an eyes-free interface for blind riders as well as an interface for sighted riders to enter information, perhaps as they stand waiting at the bus stop. 

We are working to launch a study of this system in the upcoming weeks, and we would love to have help from the community here. We are currently preparing for the first phase of the study, which involves creating the interface for OBA users to enter stop information, as described in the latter part of my system description above.  We're not yet sure if we can test with an uncapped number of people (it depends on Human Subjects), but at the very least we'd like to integrate this with OBA and perhaps have the beta testers participate.  

If you guys would like more detail, I'd be happy to elaborate! For now, here are two things I'd like to work on, and it would be great to have your guys' feedback. 
- For now, since this is an experimental study, we are creating a web app that is platform-independent to serve as the interface for entering information about a stop.  We'd like to place a button or some other type of UI element into OBA iPhone that links to this app in a spot that is noticeable by both sighted and blind people.  That is, it should probably be near the top of the screen in order to be encountered by VoiceOver, which selects UI elements starting at the top left of the screen.  It should have a name that is descriptive yet concise. We would like to have one for each StopView, so that the stop number is automatically placed on the form. 
- If possible, it'd also be nice to have an easy way of getting back to into OBA, since it is going to have to open the browser and leave the app.  If anyone knows how to do that, that'd be awesome (although it is not necessary).  We should, at the very least, have some sort of way of notifying the user that it is opening their browser and leaving, from what I've heard, this isn't obvious to blind users. 
- Lastly, a way to access the auxiliary stop information from the server, which will be separate from the main OBA server.  More on this later.

Any feedback would be great! I'll keep you guys updated. Thanks!
